bindata/network/kuryr/006-controller.yaml

```diff
@@ -0,0 +1,326 @@
+# More info about the template: https://docs.openstack.org/kuryr-kubernetes/latest/installation/containerized.html#generating-kuryr-resource-definitions-for-kubernetes
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kuryr-controller
+  namespace: kuryr
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the kuryr-controller component.
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: kuryr-controller
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: kuryr
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      initContainers:
+      - name: init-kuryr-networks
+        image: {{ .ControllerImage }}
+        command:
+        - 'python'
+        - '-c'
+        - |
+          import os
+
+          from kuryr.lib import utils
+          import netaddr
+          from neutronclient.common import exceptions as n_exc
+          import requests
+
+          from kuryr_kubernetes import config
+
+          svc_net_cidr = '{{ .ServiceSubnetCIDR }}'
```

Comment:
This could change at any time, but the InitContainer won't be re-triggered.

One way to solve this would be to move this code in to the network operator. Does that seem possible?

Reply:
Just for the record, we've agreed to implement this in the operator itself with gophercloud. This should be fairly straightforward.